idflakies-maven-plugin/src/main/java/edu/illinois/cs/dt/tools/plugin/DetectorMojo.java

```diff
@@ -195,20 +194,31 @@ public int moduleRounds(String coordinates) throws IOException {
 
     @Override
     public void execute() {
-        super.execute();
+        superExecute();
         this.outputPath = PathManager.detectionResults();
 
         final ErrorLogger logger = new ErrorLogger();
         this.coordinates = mavenProject.getGroupId() + ":" + mavenProject.getArtifactId() + ":" + mavenProject.getVersion();
 
+        long startTime = System.currentTimeMillis();
         logger.runAndLogError(() -> detectorExecute(logger, mavenProject, moduleRounds(coordinates)));
+        timing(startTime);
     }
 
-    private Void detectorExecute(final ErrorLogger logger, final MavenProject mavenProject, final int rounds) throws IOException {
+    protected void superExecute() {
```

Comment:
Is there a reason to make dedicated method that just calls one other method `ssuper.execute()`?